docs/docs/blueprint_transaction.md

```diff
@@ -0,0 +1,193 @@
+---
+sidebar_position: 99
+---
+
+# Commit using blueprint transaction
+
+ This is a small walk-through on how to use `cardano-cli` to assemble everything needed to commit funds to a `Head` using blueprint transaction.
+
+ Example assumes you have the hydra-node repo at your disposal together with `hydra-node`, `hydra-tui`, `cardano-cli` and `curl` binaries.
+
+ We can use `cardano-cli` to create a _blueprint_ transaction from some `UTxO` we own.
+
+ First we need a running cardano-node so let's spin one on the preprod network:
+
+
+ ```shell
+ ./testnets/cardano-node.sh ~/code/hydra/testnets/preprod
+ ```
+
+ Now we need to find the `UTxO` you want to commit to the `Head`
+
+ In this example we will use Alice and her external wallet key so first let's find out the address:
+
+ ```shell
+ cardano-cli address build \
+  --payment-verification-key-file hydra-cluster/config/credentials/alice-funds.vk \
+  --testnet-magic 1
+
+addr_test1vp5cxztpc6hep9ds7fjgmle3l225tk8ske3rmwr9adu0m6qchmx5z
+ ```
+
+and query to see what `UTxO` Alice has:
+
+```shell
+cardano-cli query utxo \
+   --socket-path testnets/preprod/node.socket \
+   --address addr_test1vp5cxztpc6hep9ds7fjgmle3l225tk8ske3rmwr9adu0m6qchmx5z \
+   --testnet-magic 1 \
+   --output-json
+
+{
+    "14ab373afb1112d925b0f6a84518ac26d4a8cfcc99231e1f47e6996182e843a9#0": {
+        "address": "addr_test1vp5cxztpc6hep9ds7fjgmle3l225tk8ske3rmwr9adu0m6qchmx5z",
+        "datum": null,
+        "datumhash": null,
+        "inlineDatum": null,
+        "referenceScript": null,
+        "value": {
+            "lovelace": 8000000
+        }
+    },
+    "14ab373afb1112d925b0f6a84518ac26d4a8cfcc99231e1f47e6996182e843a9#1": {
+        "address": "addr_test1vp5cxztpc6hep9ds7fjgmle3l225tk8ske3rmwr9adu0m6qchmx5z",
+        "datum": null,
+        "datumhash": null,
+        "inlineDatum": null,
+        "referenceScript": null,
+        "value": {
+            "lovelace": 1828427
+        }
+    },
+}
+```
+
+Let's pick the first `UTxO` which has total of 8 ADA available. Let's use 5 ADA to commit and rely on `hydra-node` to balance the commit transaction.
+
+```shell
+cardano-cli transaction build-raw \
+  --babbage-era \
+  --tx-in 14ab373afb1112d925b0f6a84518ac26d4a8cfcc99231e1f47e6996182e843a9#0 \
+  --tx-out addr_test1vp5cxztpc6hep9ds7fjgmle3l225tk8ske3rmwr9adu0m6qchmx5z+5000000 \
+  --fee 0 \
+  --out-file tx.json
+```
+
+We will also need pre-calculated tx-id:
+
+```shell
+cardano-cli transaction txid --tx-body-file tx.json
+
+3e94dc4236b76cfc543e662e80b93adfa11aa9b75535dad2d0c8d59963b52f9c
+```
```

Comment:
Should: also work without knowing the txid and not including `txId` in the http request.